.changeset/lemon-impalas-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+"comet-site-v8": minor
+---
+
+BREAKING: Don't hardcode values in prelogin secret and don't generate it if it would be empty. They have to be filled with Values.prelogin.secrets
```

Comment:
how can a breaking be a minor change :sweat_smile:

Reply:
We can't make major changes with the current changeset setup.
Because this chart isn't in use right now, we decided to make the breaking change as a minor release.

It's the same as here. I just forgot the prelogin secret there: https://github.com/vivid-planet/comet-charts/pull/41

Reply:
It is ok for me but the solution is not very handsome